[docs] Update invalid Vale's core settings

amandeepmittal opened the pull request:
# Why

<!--
Please describe the motivation for this PR, and link to relevant GitHub issues, forums posts, or feature requests.
-->

With the latest Vale version (`3.7.0`), some of the core settings that are configured inside `.vale.ini` file do not work when running `vale .` command locally.

For example:

![CleanShot 2024-08-22 at 16 40 22@2x](https://github.com/user-attachments/assets/18bcc7b1-6c16-4333-add8-0c2a6d1670e1)

The `IgnoredClasses` and `SkippedScopes` break the configuration and hence running the command since they are defined after the syntax `[**.*]` (which is used to define the file extension for Vale to watch for lint).

# How

<!--
How did you build this feature or fix this bug and why?
-->

FIx the Vale configuration by re-arranging the configuration settings  (`IgnoredClasses` and `SkippedScopes`) before defining the file format in `.vale.ini` file.

# Test Plan

<!--
Please describe how you tested this change and how a reviewer could reproduce your test, especially if this PR does not include automated tests! If possible, please also provide terminal output and/or screenshots demonstrating your test/reproduction.
-->

After updating the configuration in `.vale.ini`, run `vale .` or `yarn run lint-prose` locally from a terminal and there should be no errors related to invalid configuration.

# Checklist

<!--
Please check the appropriate items below if they apply to your diff. This is required for changes to Expo modules.
-->

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

amandeepmittal committed:
[docs] Update Vale's core settings

Simek reviewed (approved):
Thanks for fixing that! 🙏

amandeepmittal committed:
[docs] Enable Ellipses rule for Vale (#31116)

# Why

<!--
Please describe the motivation for this PR, and link to relevant GitHub
issues, forums posts, or feature requests.
-->

This PR removes the temporarily ignored Ellipses rule from `.vale.ini`
config file.

Switching this rule on, gives two warnings in two separate documents:

![CleanShot 2024-08-22 at 16 48
04@2x](https://github.com/user-attachments/assets/943d41ae-241c-4f38-8480-3f5a61a58ef4)

This PR fixes those two instances as well to use three dots instead of
ellipses. (Alternatively, we can also switch off Vale rule for these
code blocks, if required).

# Test Plan

<!--
Please describe how you tested this change and how a reviewer could
reproduce your test, especially if this PR does not include automated
tests! If possible, please also provide terminal output and/or
screenshots demonstrating your test/reproduction.
-->

After running fixing the Vale warnings, run `yarn run lint-prose` or
`vale .` locally to see there are no warnings for `expo-docs.Ellipses`.

# Checklist

<!--
Please check the appropriate items below if they apply to your diff.
This is required for changes to Expo modules.
-->

- [x] Documentation is up to date to reflect these changes (eg:
https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style
Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build
(eg: updated a module plugin).
